features/features.json

```diff
@@ -1207,5 +1207,17 @@
     "file": "focus-mode",
     "tags": [],
     "type": ["Website", "Theme"]
+  },
+    {
+    "title": "ChatGPT Ideas",
+    "description": "Get ideas for projects with ChatGPT on /ideas page!",
+    "credits": ["justablock", "MubiLop"],
+    "urls": [
+      "https://scratch.mit.edu/users/justablock",
+      "https://scratch.mit.edu/users/mubilop/"
+    ],
+    "file": "gpt-ideas",
```

Comment:
This should be changed to `gpt-ideas.js` and should be nested under scripts, in which it's also good to include a `runOn`...

Reply:
but other projects have without .js and my code works fine(I tested it by downloading extension localy)

Reply:
Ive just realised, this should be using v2, please read docs